src/pds/api_client/test/integration/test_classes.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from pds.api_client import Configuration
+from pds.api_client import ApiClient
+
+from pds.api_client.apis.paths.classes_class_identifier_members import ClassesClassIdentifierMembers
+
+class MyTestCase(unittest.TestCase):
+
+    def setUp(self):
+        # create an instance of the API class
+        configuration = Configuration()
+        configuration.host = 'http://localhost:8080'
+        api_client = ApiClient(configuration)
+        self.classes_members = ClassesClassIdentifierMembers(api_client)
+
+    def test_collection_members(self):
+        results = self.classes_members.get(
+            path_params={'class': 'collections', 'identifier': 'urn:nasa:pds:insight_rad:data_derived::7.0'},
+            accept_content_types=('application/json',)
+        ).body
+
+        self.assertEqual(True, False)  # add assertion here
```

Comment:
This is a mistake, let me remove that

Reply:
Note that the tests fail anyway because of the bug https://github.com/NASA-PDS/pds-api/issues/230 .

I think I am going to 'block' this ticket. That is safer than merging something which has not been tested properly.

Reply:
I made the test work, whatever it took